InWorldz/InWorldz.Region.Data.Thoosa/Serialization/PrimShapeSnapshot.cs

```diff
@@ -214,6 +214,9 @@ public byte[] SculptData
         [ProtoMember(55)]
         public int LowestLODBytes;
 
+        [ProtoMember(56)]
+        public byte[] RenderMaterials;
+
```

Comment:
Since RenderMaterials is already a protocontract, it would be more efficient to just use that class here rather than serializing first